### PR TITLE
Add CI workflows to configure labels and label pull-requests

### PR DESCRIPTION
**Pull request by wmontwe**
## Description

Add two new workflows:

* d4l-ci-label-configuration.yml: maintains GitHub labels for this project -> see `.worklow/labels.yml`
* d4l-ci-pr-labeler.yml: GitHub Action to assign labels based on folders in the project -> see `.worklow/labeler.yml`

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# meta
+- name: blocked
+  description: This is blocked
+  color: f7ae7e
+- name: duplicate
+  description: This issue or pull request already exists
+  color: cfd3d7
+- name: invalid
+  description: This doesn't seem right
+  color: cfd3d7
```
Comment (bitPogo) [inline .github/labels.yml:47]: Maybe a more alarming color like #ff088c?

**Review by bitPogo — COMMENTED** [review]
Looks in general ok for me, however I have some suggestions for more refined colors

File: .github/labels.yml (hunk 1 above)
Comment (wmontwe) [reply .github/labels.yml:50]: 👍

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  color: 64b5f6
+- name: feature
+  description: A new feature!
+  color: 81c784
+- name: good first issue
+  description: Good for newcomers
+  color: ffd54f
+- name: help wanted
+  description: Extra attention is needed
+  color: ffd54f
```
[the remark this answers is not in the document]
Comment (wmontwe) [reply .github/labels.yml:27]: 👍

File: .github/labels.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  color: 000080
+- name: auth
+  description: Auth SDK changes
+  color: 000080
+- name: crypto
+  description: Crypto SDK changes
+  color: 000080
+- name: securestore
+  description: Securestore SDK changes
+  color: 000080
```
Comment (bitPogo) [inline .github/labels.yml:81]: I am wondering, if we want to differentiate the domains by color, like for the clients...but it is in my eyes no blocker.

File: .github/labels.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
+- name: bug
+  description: Something isn't working
+  color: d73a4a
+- name: build
+  description: CI/CD or build tools
```
[the remark this answers is not in the document]
Comment (wmontwe) [reply .github/labels.yml]: I divided it in CI and build

File: .github/labels.yml [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  color: 000080
+
+
+# Sample Apps
+- name: sample-android
+  description: SampleApp Android
+  color: 4514b7
+- name: sample-jvm
+  description: SampleApp JVM
+  color: 4514b7
```
Comment (bitPogo) [inline .github/labels.yml:90]: I am wondering, if we want to differentiate the domains by color, like for the clients...but it is in my eyes no blocker.

**Review by bitPogo — COMMENTED** [review]
It looks good to me, but I have 2 non blocking questions, which liked to be cleared before I approve this PR.

File: .github/labels.yml (hunk 3 above)
Comment (wmontwe) [reply .github/labels.yml:81]: there is already a lot of color in use and it's hard to add something that differentiates

**Review by bitPogo — APPROVED** [review]
Looks good to me

**Pull request merged**